framework/db/conditions/InConditionBuilder.php

```
@@ -60,84 +65,71 @@ public function build(ExpressionInterface $expression, array &$params = [])
             $column = $column->expression;
         }
 
-        if (is_array($values)) {
-            $rawValues = $values;
-        } elseif ($values instanceof \Traversable) {
-            $rawValues = $this->getRawValuesFromTraversableObject($values);
-        }
-
+        $rawValues = $values;
         $nullCondition = null;
         $nullConditionOperator = null;
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $nullConditionOperator. Keep variable name length under 20. (undefined)

(LongVariable)